crates/serpen/tests/fixtures/mymodule.py

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+"""Test module for relative imports."""
+
+
+def helper_function():
```

Ruff (0.11.9)

Missing return type annotation for public function `helper_function`

Add return type annotation: `str`

(ANN201)